game/management/commands/import_openings.py

```
@@ -0,0 +1,119 @@
+                                existing.name = name
+                                existing.moves = pgn
+                                existing.move_count = half_moves
+                                existing.save()
+                        except Opening.DoesNotExist:
+                            Opening.objects.create(
+                                fen=fen, eco=eco, name=name,
+                                moves=pgn, move_count=half_moves,
+                            )
+                            created += 1
+
+                except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)